packages/devtools_flutter/pubspec.yaml

```diff
@@ -0,0 +1,41 @@
+name: devtools_flutter
+description: UI package for web-based performance tooling for Dart and Flutter written in Flutter.
+
+# Note: when updating this version, please update the corresponding entry in
+# lib/devtools.dart and ../devtools/pubspec.yaml as well to the same version.
+# When publishing new versions of this package be sure to publish a new version
+# of package:devtools as well. package:devtools contains a compiled snapshot of
+# this package.
```

Comment:
does this comment hold true?

Reply:
No.  In fact, this package may end up being short-lived.